scripts/potoken_provider/setup_pot_provider.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -e
+
+SCRIPTS_DIR="scripts/potoken_provider"
+TARGET_DIR="$SCRIPTS_DIR/bgutil-provider"
+GEN_SCRIPT="$TARGET_DIR/build/generate_once.js"
+
+# Ensure the server directory exists
+if [ ! -d "$TARGET_DIR" ]; then
+    echo "Error: PO Token provider server directory is missing! Please run scripts/update_pot_provider.sh first."
+    exit 1
+fi
+
+cd "$TARGET_DIR" || exit 1
+
+# Check if dependencies need installation
+if [ ! -d "node_modules" ]; then
+    echo "Installing dependencies..."
+    yarn install --frozen-lockfile
+else
+    echo "Dependencies already installed. Skipping yarn install."
+fi
+
+# Check if build directory exists and if transpiling is needed
+if [ ! -d "build" ]; then
```

Comment:
How does this check if the build is up to date? Doesn't this only check if it has been transpiled once some indeterminate time ago?

Reply:
Curious why you don't just merge these two bash scripts together. If the `update` script also runs the dependency install, and the transpilation, you don't need these special checks to tell if something has changed